translation_center.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
   s.files = Dir["{app,config,db,lib}/**/*"] + ["MIT-LICENSE", "Rakefile", "README.md"]
   s.test_files = Dir["test/**/*"]
 
-  s.add_dependency "rails", ">= 3.1.0", "<= 3.2.12"
+  s.add_dependency "rails", ">= 3.1.0", "<= 3.2.40"
```

Comment:
Good point! However "~> 3.2" that you suggested will also catch the update to version Rails 3.3 ;-) Sure there will not be Rails 3.3, but it should rather be eg. "~> 3.2.19". I will correct it accordingly.